dist/angular-patternfly.js

```diff
@@ -698,6 +697,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', ["c3ChartDefaul
           var color = '#0088CE';
 
           if (thresholds) {
+            color = '#3f9c35';
```

Comment:
I know that these colors were already in here but I wonder if it makes more sense to add a class (`.threshold-error` and `.threshold-warning`, for example) instead of setting these colors directly.  That would allow the projects that consume angular-patternfly to override these colors if necessary.